src/ui/src/builder/settings/BuilderSettingsProperties.spec.ts

```
@@ -16,6 +16,8 @@
 import templateMap from "@/core/templateMap";
 import { useSecretsManager } from "@/core/useSecretsManager";
 import { FieldCategory, WriterComponentDefinitionField } from "@/writerTypes";
+import { WriterComponentDefinitionField } from "@/writerTypes";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'WriterComponentDefinitionField'. Consider to delete it or rename it.

'WriterComponentDefinitionField' is defined here:


(lint/suspicious/noRedeclare)